massa-bootstrap/src/tests/messages.rs

```diff
@@ -178,14 +178,6 @@ fn test_serialize_error_cases_servermsg() {
             // TODO    Remove once we generate denunciations in the block header
             continue;
         }
-        if n == 6 {
-            // TODO    Remove once limits for deser of final_state_parts is by bytes, not length
-            continue;
-        }
-        if n == 8 {
-            // TODO    Remove once limits for deser of versionning_parts is by bytes, not length
-            continue;
-        }
         println!("Testing {n} faulty case");
         let mut bytes = Vec::new();
         let msg = BootstrapServerMessage::generate_faulty(&mut rng, n);
```

Comment:
Using n with just a number is difficult to read. Maybe use an enum here? So it would be easier to read and understand why we skip some of them for now...

Reply:
Thanks, it's better indeed, done.